bindings/nodejs/lib/client/client.ts

```diff
@@ -67,14 +67,14 @@ import {
     OutputWithMetadataResponse,
     NetworkMetricsResponse,
 } from '../types/models/api';
-import { RoutesResponse } from '../types/models/api/routes-response';
+import { RoutesResponse } from '../types/models/api/node/routes-response';
```

Comment:
still `-response`?

Reply:
Overall confused about the places with/without this suffix

Reply:
I removed all the `-response` suffixes from the files and made the overall structure flat, so all response types live in their respective route group as specified by TIP-48.